### PR TITLE
sprint21/KPV-2122 :: BackerType edit/show

### PR DESCRIPTION
**Pull request by alberKuorum**
sprint21/KPV-2122  :: BackerType edit and show changes

### CODE REVIEW

File: grails-app/i18n/messages.properties [hunk 1]
```diff
@@ -1887,6 +1887,9 @@ debate.proposal.placeholder = &\#xf0eb;  Write your proposal here...
 debate.proposal.comment.placeholder = &\#xf0eb;  Write your comment here...
 debate.timesSeen = {0} times seen
 debate.authorFollowers = The author has {0} followers
+debate.backerType = {0}
+debate.backerType.PUBLIC_INSTITUTION = Public institution
+debate.backerType.CITIZEN_INSTITUTION = Citizen institution
```
Comment (iduetxe) [inline grails-app/i18n/messages.properties]: Add them to poe.
You needs other key:
debate.backerType.CITIZEN

File: grails-app/i18n/messages.properties [hunk 2]
```diff
@@ -1300,7 +1300,7 @@ kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCo
 kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.districtId.modal.button.ok = Yes, delete all the votes and supports
 kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.districtId.modal.button.cancel = Ouch, no. Let it as is.
 kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.cause.label = Cause
-kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.backerType.label = Type
+kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.backerType.label = BackerType
```
Comment (iduetxe) [inline grails-app/i18n/messages.properties]: This text should be 2 words, isn't it? -> Backer Type => Fix it on POE as well

**Review by iduetxe — CHANGES_REQUESTED** [review]

File: grails-app/i18n/messages.properties (hunk 2 above)
Comment (alberKuorum) [reply grails-app/i18n/messages.properties]: It has been changed on POE

**Review by alberKuorum — COMMENTED** [review]
Ready to review

**Review by iduetxe — CHANGES_REQUESTED** [review]
Reuse i18n codes

**Review by iduetxe — APPROVED** [review]

**Pull request merged**